recipe/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: mpl-probscale
-  version: 0.1.1
+  version: 0.1.2
 
 source:
   git_url: https://github.com/phobson/mpl-probscale.git
```

Comment:
There is a mild preference for downloading tarballs rather than using git tags - the main motivator is that the git repo can get pretty big pretty quick.